pkg/core/util.go

```diff
@@ -138,7 +138,7 @@ func CalculateNetworkFee(script []byte) (int64, int) {
 	)
 	if vm.IsSignatureContract(script) {
 		size += 67 + io.GetVarSize(script)
-		netFee += opcodePrice(opcode.PUSHDATA1, opcode.PUSHNULL) + crypto.ECDSAVerifyPrice
+		netFee += opcodePrice(opcode.PUSHDATA1, opcode.PUSHNULL, opcode.PUSHDATA1) + crypto.ECDSAVerifyPrice
```

Comment:
We need a test for it (and multisig). Calculate the network fee, then run invocation and verification scripts and sum gas consumed with per-byte fee for the length of transcation. They should perfectly match.